src/tests/sql/init_version_5.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE objectprocessorqueue;
```

Comment:
Why drop here, if you are going to create it later?

Reply:
you're right, if it's removed and there is an error, it means there is a bug somewhere